### DISCUSSION TITLE
Image Group

### DISCUSSION THREAD

**Discussion by annefranca** [Telefonica/mistica-design#1146] [Proposals]
### Goal

Proposta da Work & Co para a visualização de aplicativos inclusos.

### Context

Deixar visível para a pessoa usuária a relação de aplicativos inclusos dentro de uma Assinatura. 

### Impact

Proposta Work & Co - Novo App.

### Discussion Questions

Esse novo componente ficaria hospedado em qual biblioteca e qual equipe ficaria responsável pela documentação e implementação no Figma? 

### Do you have a visual proposal?

https://www.figma.com/file/seiNjy7YmPWEY3BfY4WYpp/Novos-componentes?node-id=1%3A9140&t=oW9fYQ4r9pTIoxru-1

**Comment by aweell** [comment]
Hola @annefranca,

Llegados al punto donde convivirían el `avatarGroup` y el `imageGroup` tendría sentido valorar un componente con un nivel de abstracción mayor (me invento un nombre: `stackingGroup`) al que le puedas pasar una imagen o un avatar y permita:

* Apilamiento de elementos
* Cambio en la dirección de apilamiento: de izquierda a derecha o viceversa
* Tamaño de los items dentro del grupo?
* Otras propiedades...

De esta manera se podrían cubrir esos dos casos de uso:

* Imágenes
* Avatares

Y cubrir posibles casos a futuro, como por ejemplo que se puedan apilar circles...

**Reply by annefranca** [reply]
Olá @aweell :)

Ok, vamos dar andamento com nessa tarefa e atualizar a documentação do Avatar Group (https://www.figma.com/file/Ln9nFcdE0PUmDuMwQYp7Yu/Avatar-Group?node-id=108%3A380&t=4lrSmGyYPxJl7P1k-1)

**Reply by annefranca** [reply]
@aweell @yceballost 

Segue a documentação inicial atualizada contemplando o Avatar e Image > Stacking Group

**Comment by yceballost** [comment]
We are going to create a new component called `StackingGroup`

https://github.com/Telefonica/mistica-design/issues/1100

**Reply by annefranca** [reply]
@yceballost esse componente ficaria como Mística Team?

**Reply by yceballost** [reply]
Vivo!

**Reply by annefranca** [reply]
@yceballost você vai liberar a Branch?

**Reply by yceballost** [reply]
yay! aquí está
https://www.figma.com/file/3ME86VKaAQ2BoeqoQuHMYx/branch/2xUQantWwtpJotRFBWM77k/%F0%9F%94%B8-StackingGroup-Specs?node-id=0%3A1&t=0UmYlYVBjhSoF9wi-1
